### ISSUE TITLE
The ckpt file does not work

### ISSUE THREAD

**Issue by tengerye** [davidsandberg/facenet#502]
I tried to find-tune the pre-trained model on my own data. I downloaded the pre-trained model from [MS-Celeb-1M](https://github.com/davidsandberg/facenet/wiki). The folder contains four files: a .pb file, a .ckpt.data file, a .ckpt.index file, and a .meta file. 

I tried to [restore](https://github.com/davidsandberg/facenet/blob/master/src/train_tripletloss.py) the model from the last three files but got the same error:

> DataLossError (see above for traceback): Unable to open table file ../models/20170512-110547/model-20170512-110547.meta: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

Does the .ckpt file not correspond to .pb file?
Thank you very much in advance.

**Comment by ronyuzhang** [comment]
i also encountered the same problem, and how to solve this about you?

**Comment by tengerye** [comment]
I managed to fix it. Basically, my solution is based on [this answer](https://stackoverflow.com/questions/41265035/tensorflow-why-there-are-3-files-after-saving-the-model). The TensorFlow does not produce .ckpt file but separated three files. The option to run the train_tripletloss.py is `--pretrained_model ../models/20170512-110547/model-20170512-110547.ckpt-250000`.

Hope it could help you.
